### ISSUE TITLE
timestamp out of range for platform localtime() but still submitted to the database!

### ISSUE THREAD

**Issue by Bernardrouhi** [mogui/pyorient#232]
when I try to add a record to Book V class by using this:  
`client.command("INSERT INTO Books set name='Candide2', published_date='1700-1-1'")`  

I get `ValueError: timestamp out of range for platform localtime() function` but yet it is still submitted to the database!  
and when I try to query from rid by using     
`print client.command("SELECT FROM Books WHERE @rid = #43:0")[0].__dict__['_OrientRecord__o_storage']['published_date']`, 

it's work perfetcly fine.

However, if I try to add another record but I give time like so:
`client.command("INSERT INTO Books set name='Candide3', published_date='1700-1-1 02:00:00'")`
again it will throw back ValueError as well as even when I try to query. 

I think that would be great if we could just get a string like how we pass the date, instead of `datetime` type.

**Comment by anber500** [comment]
What if you try the following?

    client.command("INSERT INTO Books set name='Candide3', published_date=DATE('1700-01-01 02:00:00')")

Notice the "DATE" parameter encapsulating the date string. Your error should disappear. 

As far as returning a string instead of a date, I'm not sure that is a good idea. It's far simpler to convert a date to a string than to a meaningful python object. Especially if you have to consider timezones. Also you can't do anything with string dates. You'll have to convert them to `datetime` anyways before you can use it.

**Comment by Bernardrouhi** [comment]
Yeah, that did work out, thanks for the solution, that should be mention in a documentation.

well, since I send it as a string I expect it to return back as a string too, but now I'm thinking about it, you are right, having a meaningful type make the workflow easier, but again it would be good to have documentation about it, so we understand how we should and should not use them, otherwise with simple mistake we might think the package is unstable and we shouldn't rely on it.
